src/helpers/api.ts

```diff
@@ -58,7 +59,7 @@ class Api {
         return this._request(url, { ...options, method: 'patch' });
     }
 
-    private _request(url: string, options: AbstractMethodRequestOptions): Promise<void> {
+    private _request(url: string, options: AbstractMethodRequestOptions): Promise<any> {
```

Comment:
Т.к. эта функция возвращает Promise, она является асинхронной. Лучше явно это указать, потому что явное лучше, чем не явное. Код должен быть настолько понятным, что при первом взгляде на него я должен понять, что тут происходит
```suggestion
    private async _request(url: string, options: AbstractMethodRequestOptions): Promise<any> {
```